docs/contributing/standards.md

```diff
@@ -6,38 +6,35 @@ sidebar_label: Standards
 
 # Standards
 
-The following standards are used by FAST-DNA contributors for consistency across this repository.
+Our contributors use these standards for consistency and quality in the repository.
 
-## Naming Conventions
+## Naming conventions
 
-* Avoid all abbreviations, except where common industry acronyms.
+* Avoid abbreviations unless they are common industry acronyms.
 * Use names that are clear and intuitive.
 * Use semantic names instead of presentational.
 
 ## Accessibility
 
-FAST-DNA works diligently from the ground up to enable accessibility in collaboration with legal, accessibility, and marketing teams. As an ecosystem to accelerate the building of website's and apps we ensure you have the tools to build accessible user exerpiences.
+Acknowledge accessibility from the start of each new feature in collaboration with the legal and accessibility. As a system for the building of websites and applications, we ensure you have the tools to create accessible user experiences.
```

Comment:
"Acknowledge accessibility from the start of each new feature in collaboration with the legal and accessibility." reads a little strangely. The legal and accessibility what? team?

Reply:
Fixed.